spec/support/fixtures/police/pdf-text-all-no-answers.txt

```diff
@@ -1,5 +1,5 @@
 Outbound vehicle
-Registration                     Cell number                        PECS Barcode
+Registration                     Cell number                       PECS Barcode
```

Comment:
I just eyeballed these changes.

Assuming its related to PDF parsing
not PDF generation.

Don't know how you normally handle that.